Core/include/Acts/EventData/SingleFreeTrackParameters.hpp

```diff
@@ -169,10 +167,10 @@ class SingleFreeTrackParameters {
   ///
   /// @note The context is not used here but makes the API consistent with
   /// @c SingleCurvilinearTrackParameters and @c SingleBoundTrackParameters
-  template <unsigned int par,
+  template <FreeParametersIndices par,
```

Comment:
Naming: as above. `par` should be `kIndex` or `kIdx`.

Reply:
done